server/test/unit/server/content/test_storage.py

```diff
@@ -62,11 +62,14 @@ def test_exit(self):
 
 class TestFileStorage(TestCase):
 
+    @patch('pulp.server.content.storage.sha256')
```

Comment:
FWIW I don't think there's much value in mocking out standard library calls unless they do something expensive like hit disk, network, etc.

And in this case, since we're depending on the string representation of a list being consistent among python implementations, it would actually be valuable to hard-code the expected real hash, and compare it against the calculated hash.